### ISSUE TITLE
Crash on server load because of InjectableRecipes.getAllRecipes

### ISSUE THREAD

**Issue by TiphaineLAURENT** [orangemonkey68/injectable-recipes#1]
I got this crash when trying to create a new world. The code is the one you provided in your [example](https://github.com/orangemonkey68/injectable-recipes/blob/master/src/main/java/me/orangemonkey68/injectablerecipes/example/ExampleInit.java).

```java
Failed to load datapacks, can't proceed with server load
 java.util.concurrent.ExecutionException: java.lang.NoSuchMethodError: 'java.util.Map me.orangemonkey68.injectablerecipes.InjectableRecipes.getAllRecipes()'
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395) ~[?:?]
CompletableFuture.java:395
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2063) ~[?:?]
CompletableFuture.java:2063
	at net.minecraft.client.MinecraftClient.method_29604(MinecraftClient.java:1846) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
MinecraftClient.java:1846
	at net.minecraft.client.MinecraftClient.startIntegratedServer(MinecraftClient.java:1680) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
MinecraftClient.java:1680
	at net.minecraft.client.MinecraftClient.method_29607(MinecraftClient.java:1664) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
MinecraftClient.java:1664
	at net.minecraft.client.gui.screen.world.CreateWorldScreen.createLevel(CreateWorldScreen.java:270) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
CreateWorldScreen.java:270
	at net.minecraft.client.gui.screen.world.CreateWorldScreen.method_19922(CreateWorldScreen.java:214) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
CreateWorldScreen.java:214
	at net.minecraft.client.gui.widget.ButtonWidget.onPress(ButtonWidget.java:26) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
ButtonWidget.java:26
	at net.minecraft.client.gui.widget.AbstractPressableButtonWidget.onClick(AbstractPressableButtonWidget.java:17) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
AbstractPressableButtonWidget.java:17
	at net.minecraft.client.gui.widget.AbstractButtonWidget.mouseClicked(AbstractButtonWidget.java:136) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
AbstractButtonWidget.java:136
	at net.minecraft.client.gui.ParentElement.mouseClicked(ParentElement.java:42) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
ParentElement.java:42
	at net.minecraft.client.Mouse.method_1611(Mouse.java:85) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
Mouse.java:85
	at net.minecraft.client.gui.screen.Screen.wrapScreenError(Screen.java:429) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
Screen.java:429
	at net.minecraft.client.Mouse.onMouseButton(Mouse.java:85) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
Mouse.java:85
	at net.minecraft.client.Mouse.method_22686(Mouse.java:172) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
Mouse.java:172
	at net.minecraft.util.thread.ThreadExecutor.execute(ThreadExecutor.java:86) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
ThreadExecutor.java:86
	at net.minecraft.client.Mouse.method_22684(Mouse.java:172) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36) [lwjgl-glfw-3.2.2.jar:build 10]
GLFWMouseButtonCallbackI.java:36
	at org.lwjgl.system.JNI.invokeV(Native Method) ~[lwjgl-3.2.2.jar:build 10]
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3174) [lwjgl-glfw-3.2.2.jar:build 10]
GLFW.java:3174
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:112) [minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
RenderSystem.java:112
	at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1006) [minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
MinecraftClient.java:1006
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:601) [minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
MinecraftClient.java:601
	at net.minecraft.client.main.Main.main(Main.java:187) [minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
Main.java:187
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
NativeMethodAccessorImpl.java:62
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
DelegatingMethodAccessorImpl.java:43
	at java.lang.reflect.Method.invoke(Method.java:564) ~[?:?]
Method.java:564
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192) [fabric-loader-0.9.3+build.207@net.fabricmc.yarn.1_16_3.1.16.3+build.1-v2.jar:?]
MinecraftGameProvider.java:192
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:140) [fabric-loader-0.9.3+build.207@net.fabricmc.yarn.1_16_3.1.16.3+build.1-v2.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26) [fabric-loader-0.9.3+build.207@net.fabricmc.yarn.1_16_3.1.16.3+build.1-v2.jar:?]
KnotClient.java:26
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86) [dev-launch-injector-0.2.1+build.8.jar:?]
Caused by: java.lang.NoSuchMethodError: 'java.util.Map me.orangemonkey68.injectablerecipes.InjectableRecipes.getAllRecipes()'
	at net.minecraft.recipe.RecipeManager.handler$zei000$injectRecipes(RecipeManager.java:1030) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
RecipeManager.java:1030
	at net.minecraft.recipe.RecipeManager.apply(RecipeManager.java:72) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
RecipeManager.java:72
	at net.minecraft.recipe.RecipeManager.apply(RecipeManager.java:46) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
RecipeManager.java:46
	at net.minecraft.resource.SinglePreparationResourceReloadListener.method_18790(SinglePreparationResourceReloadListener.java:13) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
SinglePreparationResourceReloadListener.java:13
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:714) ~[?:?]
CompletableFuture.java:714
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:478) ~[?:?]
CompletableFuture.java:478
	at net.minecraft.resource.ResourceReloader.method_18365(ResourceReloader.java:69) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
at net.minecraft.util.thread.ThreadExecutor.executeTask(ThreadExecutor.java:136) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
ThreadExecutor.java:136
	at net.minecraft.util.thread.ReentrantThreadExecutor.executeTask(ReentrantThreadExecutor.java:22) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
ReentrantThreadExecutor.java:22
	at net.minecraft.util.thread.ThreadExecutor.runTask(ThreadExecutor.java:109) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
ThreadExecutor.java:109
	at net.minecraft.util.thread.ThreadExecutor.runTasks(ThreadExecutor.java:119) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
ThreadExecutor.java:119
	at net.minecraft.client.MinecraftClient.method_29604(MinecraftClient.java:1845) ~[minecraft-1.16.3-mapped-net.fabricmc.yarn-1.16.3+build.1-v2.jar:?]
MinecraftClient.java:1845
	... 29 more
```

**Comment by orangemonkey68** [comment]
In your build.gradle have you included the mod?

**Comment by TiphaineLAURENT** [comment]
Yes. I have `modImplementation "me.orangemonkey68:injectable-recipes:1.0.2"` in the dependencies section and I added `repositories {
	maven { url = "https://dl.bintray.com/orangemonkey68/mods" }
}`

**Comment by orangemonkey68** [comment]
Add `include "me.orangemonkey68:injectable-recipes:1.0.2"` and see if the crash persists

**Comment by TiphaineLAURENT** [comment]
I now have both
```java
	include "me.orangemonkey68:injectable-recipes:1.0.2"
	modImplementation "me.orangemonkey68:injectable-recipes:$1.0.2"
```
in the dependencies section but I still have the same bug.

**Comment by orangemonkey68** [comment]
I just whipped up a project and got the same issue. I'm looking into it.

**Comment by orangemonkey68** [comment]
Bug should be fixed with v1.0.3
